p2p/src/authenticated/actors/tracker/actor.rs

```diff
@@ -165,7 +170,7 @@ impl<E: Spawner + Rng + Clock + GClock, C: Scheme> Actor<E, C> {
             .expect("failed to get current time")
             .as_secs();
         let (socket_bytes, payload_bytes) = socket_peer_payload(&cfg.address, current_time);
-        let ip_signature = cfg.crypto.sign(NAMESPACE, &payload_bytes);
+        let ip_signature = cfg.crypto.sign(&suffix_namespace(&cfg.namespace), &payload_bytes);
```

Comment:
I recommend storing `suffix_namespace(&cfg.namespace)` as `namespace` on the struct. The current approach requires allocating memory during each `sign`/`verify`.